.gitignore

```diff
@@ -0,0 +1,2 @@
+text.md
```

Comment:
- [ ] Kindly fix the mis-spelling for the `text.md` file as it should be named `test.md`.